src/raghvendra/BankOperationUsingSwitch.java

```diff
@@ -0,0 +1,153 @@
+package raghvendra;
+
+import java.util.Scanner;
+
+public class BankOperationUsingSwitch {
+	static int creditCount;
+	static int debitCount;
+	static double hdfcAmount;
+	static int hdfcCreditCount;
+	static int hdfcDebitCount;
+	static int iciciDebitCount;
+	static int sbiDebitCount;
+	static int bobDebitCount;
+	static double iciciAmount;
+	static int iciciCreditCount;
+	static double bobAmount;
+	static int bobCreditCount;
+	static double sbiAmount;
+	static int sbiCreditCount;
+	static double creditAmount;
+	static double debitAmount;
+	static double totalBalance;
+	public void bankSetInitialBalance() {
+		System.out.println("\r\n"+"Enter HDFC Initial amount");
+		Scanner sc = new Scanner(System.in);
+		hdfcAmount=sc.nextDouble();
+		System.out.println("Enter ICICI Initial amount");
+		iciciAmount=sc.nextDouble();
+		System.out.println("Enter BOB Initial amount");
+		bobAmount=sc.nextDouble();
+		System.out.println("Enter SBI Initial amount");
+		sbiAmount=sc.nextDouble();
+		System.out.println("Enter Bank Name");
+		String bankName=sc.next();
+		System.out.println("Enter Operation");
+		String operation=sc.next();
+		if (operation.equalsIgnoreCase("Credit")) {
+			System.out.println("Enter Credit Amount");
+			creditAmount=sc.nextDouble();
+		}
+		else {
+			System.out.println("Enter Debit Amount");
+			debitAmount=sc.nextDouble();
+		}
+	
+		bankSwitch(bankName, operation, creditAmount, debitAmount);
+		bankBalance(bankName);
+		bankCreditDebit();
+	}
+	
+
+	public void bankSwitch(String bankName,String operation,double creditAmount,double debitAmount) {
```

Comment:
Hi @RaghvendramaniTOct2020 
Here think on real time scenario when we are going to bank for credit operation, that time bank not asking customer to do debit as well right?

In this method, it give impression like above scenario.

Reply:
incorporated the changes suggested. Thanks